### ISSUE TITLE
cairo.h not found Arch Linux

### ISSUE THREAD

**Issue by DarkStar1997** [openframeworks/openFrameworks#6202]
I downloaded the nightly build for openframeworks from 
[linux64gcc6_nightly](https://openframeworks.cc/ci_server/versions/nightly/of_v20181222_linux64gcc6_nightly.tar.gz)
The compilation of 3d/cameralensOffsetExample produces the following output:
```
[rohan@rohan-pc cameraLensOffsetExample]$ make -j8
HOST_OS=Linux
checking pkg-config libraries:   cairo zlib gstreamer-app-1.0 gstreamer-1.0 gstreamer-video-1.0 gstreamer-base-1.0 libudev freetype2 fontconfig sndfile openal libcurl glfw3 rtaudio libpulse-simple alsa gl glu glew gtk+-3.0 libmpg123 
with PKG_CONFIG_LIBDIR=
Compiling OF library for Release
make[1]: Entering directory '/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworksCompiled/project'
HOST_OS=Linux
checking pkg-config libraries:   cairo zlib gstreamer-app-1.0 gstreamer-1.0 gstreamer-video-1.0 gstreamer-base-1.0 libudev freetype2 fontconfig sndfile openal libcurl glfw3 rtaudio libpulse-simple alsa gl glu glew gtk+-3.0 libmpg123 
with PKG_CONFIG_LIBDIR=
HOST_OS=Linux
checking pkg-config libraries:   cairo zlib gstreamer-app-1.0 gstreamer-1.0 gstreamer-video-1.0 gstreamer-base-1.0 libudev freetype2 fontconfig sndfile openal libcurl glfw3 rtaudio libpulse-simple alsa gl glu glew gtk+-3.0 libmpg123 
with PKG_CONFIG_LIBDIR=
HOST_OS=Linux
checking pkg-config libraries:   cairo zlib gstreamer-app-1.0 gstreamer-1.0 gstreamer-video-1.0 gstreamer-base-1.0 libudev freetype2 fontconfig sndfile openal libcurl glfw3 rtaudio libpulse-simple alsa gl glu glew gtk+-3.0 libmpg123 
with PKG_CONFIG_LIBDIR=
Done!
make[1]: Leaving directory '/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworksCompiled/project'


Compiling cameraLensOffsetExample for Release
make[1]: Entering directory '/home/rohan/Downloads/of_v20181222_linux64gcc6_release/examples/3d/cameraLensOffsetExample'
HOST_OS=Linux
checking pkg-config libraries:   cairo zlib gstreamer-app-1.0 gstreamer-1.0 gstreamer-video-1.0 gstreamer-base-1.0 libudev freetype2 fontconfig sndfile openal libcurl glfw3 rtaudio libpulse-simple alsa gl glu glew gtk+-3.0 libmpg123 
with PKG_CONFIG_LIBDIR=
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
Package 'opencv', required by 'virtual:world', not found
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
Package 'opencv', required by 'virtual:world', not found
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
Package 'opencv', required by 'virtual:world', not found
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
Package 'opencv', required by 'virtual:world', not found
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
Package 'opencv', required by 'virtual:world', not found
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
Package 'opencv', required by 'virtual:world', not found
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
Package 'opencv', required by 'virtual:world', not found
Compiling /home/rohan/Downloads/of_v20181222_linux64gcc6_release/examples/3d/cameraLensOffsetExample/src/ofApp.cpp
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
Package 'opencv', required by 'virtual:world', not found
g++ -c -O3 -march=native -mtune=native -DNDEBUG -Wall -Werror=return-type -std=c++14 -DGCC_HAS_REGEX -DOF_USING_GTK -DOF_USING_GTK -DOF_USING_MPG123  -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/fmodex/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm/simd -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm/gtc -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm/gtx -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm/detail -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/json/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/kiss/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/pugixml/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/tess2/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/utf8/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/utf8/include/utf8 -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/sound -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/app -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/communication -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/types -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/math -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/events -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/3d -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/video -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/graphics -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/gl -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/utils -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/examples/3d/cameraLensOffsetExample/src -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/ofxOpenCv/src -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/ofxOpenCv/libs -MMD -MP -MF obj/linux64/Release/src/ofApp.d -MT obj/linux64/Release/src/ofApp.o -o obj/linux64/Release/src/ofApp.o -c /home/rohan/Downloads/of_v20181222_linux64gcc6_release/examples/3d/cameraLensOffsetExample/src/ofApp.cpp
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
Package 'opencv', required by 'virtual:world', not found
Compiling /home/rohan/Downloads/of_v20181222_linux64gcc6_release/examples/3d/cameraLensOffsetExample/src/main.cpp
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
Package 'opencv', required by 'virtual:world', not found
g++ -c -O3 -march=native -mtune=native -DNDEBUG -Wall -Werror=return-type -std=c++14 -DGCC_HAS_REGEX -DOF_USING_GTK -DOF_USING_GTK -DOF_USING_MPG123  -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/fmodex/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm/simd -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm/gtc -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm/gtx -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm/detail -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/json/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/kiss/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/pugixml/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/tess2/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/utf8/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/utf8/include/utf8 -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/sound -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/app -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/communication -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/types -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/math -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/events -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/3d -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/video -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/graphics -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/gl -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/utils -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/examples/3d/cameraLensOffsetExample/src -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/ofxOpenCv/src -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/ofxOpenCv/libs -MMD -MP -MF obj/linux64/Release/src/main.d -MT obj/linux64/Release/src/main.o -o obj/linux64/Release/src/main.o -c /home/rohan/Downloads/of_v20181222_linux64gcc6_release/examples/3d/cameraLensOffsetExample/src/main.cpp
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
Package 'opencv', required by 'virtual:world', not found
Compiling /home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/ofxOpenCv/src/ofxCvHaarFinder.cpp
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
Package 'opencv', required by 'virtual:world', not found
g++ -c -O3 -march=native -mtune=native -DNDEBUG -Wall -Werror=return-type -std=c++14 -DGCC_HAS_REGEX -DOF_USING_GTK -DOF_USING_GTK -DOF_USING_MPG123  -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/fmodex/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm/simd -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm/gtc -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm/gtx -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm/detail -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/json/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/kiss/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/pugixml/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/tess2/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/utf8/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/utf8/include/utf8 -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/sound -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/app -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/communication -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/types -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/math -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/events -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/3d -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/video -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/graphics -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/gl -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/utils -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/ofxOpenCv/src -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/ofxOpenCv/libs -MMD -MP -MF /home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/obj/linux64/Release/ofxOpenCv/src/ofxCvHaarFinder.d -MT /home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/obj/linux64/Release/ofxOpenCv/src/ofxCvHaarFinder.o -o /home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/obj/linux64/Release/ofxOpenCv/src/ofxCvHaarFinder.o -c /home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/ofxOpenCv/src/ofxCvHaarFinder.cpp
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
Package 'opencv', required by 'virtual:world', not found
Compiling /home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/ofxOpenCv/src/ofxCvColorImage.cpp
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
Package 'opencv', required by 'virtual:world', not found
g++ -c -O3 -march=native -mtune=native -DNDEBUG -Wall -Werror=return-type -std=c++14 -DGCC_HAS_REGEX -DOF_USING_GTK -DOF_USING_GTK -DOF_USING_MPG123  -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/fmodex/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm/simd -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm/gtc -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm/gtx -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/glm/include/glm/detail -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/json/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/kiss/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/pugixml/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/tess2/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/utf8/include -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/utf8/include/utf8 -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/sound -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/app -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/communication -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/types -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/math -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/events -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/3d -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/video -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/graphics -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/gl -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/utils -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/ofxOpenCv/src -I/home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/ofxOpenCv/libs -MMD -MP -MF /home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/obj/linux64/Release/ofxOpenCv/src/ofxCvColorImage.d -MT /home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/obj/linux64/Release/ofxOpenCv/src/ofxCvColorImage.o -o /home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/obj/linux64/Release/ofxOpenCv/src/ofxCvColorImage.o -c /home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/ofxOpenCv/src/ofxCvColorImage.cpp
In file included from /home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/ofxOpenCv/src/ofxCvImage.h:15:0,
                 from /home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/ofxOpenCv/src/ofxCvGrayscaleImage.h:12,
                 from /home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/ofxOpenCv/src/ofxCvColorImage.cpp:1:
/home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/ofxOpenCv/src/ofxCvConstants.h:10:10: fatal error: cv.h: No such file or directory
 #include "cv.h"
          ^~~~~~
compilation terminated.
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
Package 'opencv', required by 'virtual:world', not found
make[1]: *** [/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworksCompiled/project/makefileCommon/compile.project.mk:360: /home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/obj/linux64/Release/ofxOpenCv/src/ofxCvColorImage.o] Error 1
make[1]: *** Waiting for unfinished jobs....
In file included from /home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/ofMain.h:65:0,
                 from /home/rohan/Downloads/of_v20181222_linux64gcc6_release/examples/3d/cameraLensOffsetExample/src/ofApp.h:3,
                 from /home/rohan/Downloads/of_v20181222_linux64gcc6_release/examples/3d/cameraLensOffsetExample/src/ofApp.cpp:1:
/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/graphics/ofCairoRenderer.h:4:10: fatal error: cairo.h: No such file or directory
 #include "cairo.h"
          ^~~~~~~~~
compilation terminated.
make[1]: *** [/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworksCompiled/project/makefileCommon/compile.project.mk:223: obj/linux64/Release/src/ofApp.o] Error 1
In file included from /home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/ofMain.h:65:0,
                 from /home/rohan/Downloads/of_v20181222_linux64gcc6_release/examples/3d/cameraLensOffsetExample/src/ofApp.h:3,
                 from /home/rohan/Downloads/of_v20181222_linux64gcc6_release/examples/3d/cameraLensOffsetExample/src/main.cpp:1:
/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/graphics/ofCairoRenderer.h:4:10: fatal error: cairo.h: No such file or directory
 #include "cairo.h"
          ^~~~~~~~~
compilation terminated.
make[1]: *** [/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworksCompiled/project/makefileCommon/compile.project.mk:223: obj/linux64/Release/src/main.o] Error 1
In file included from /home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/ofMain.h:65:0,
                 from /home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/ofxOpenCv/src/ofxCvHaarFinder.h:14,
                 from /home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/ofxOpenCv/src/ofxCvHaarFinder.cpp:1:
/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworks/graphics/ofCairoRenderer.h:4:10: fatal error: cairo.h: No such file or directory
 #include "cairo.h"
          ^~~~~~~~~
compilation terminated.
make[1]: *** [/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworksCompiled/project/makefileCommon/compile.project.mk:360: /home/rohan/Downloads/of_v20181222_linux64gcc6_release/addons/obj/linux64/Release/ofxOpenCv/src/ofxCvHaarFinder.o] Error 1
make[1]: Leaving directory '/home/rohan/Downloads/of_v20181222_linux64gcc6_release/examples/3d/cameraLensOffsetExample'
make: *** [/home/rohan/Downloads/of_v20181222_linux64gcc6_release/libs/openFrameworksCompiled/project/makefileCommon/compile.project.mk:129: Release] Error 2
[rohan@rohan-pc cameraLensOffsetExample]$ 

```

The output of pkg-config is:

```
rohan@rohan-pc cameraLensOffsetExample]$ pkg-config --cflags --libs cairo
-I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/lib/libffi-3.2.1/include -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/uuid -lcairo 
[rohan@rohan-pc cameraLensOffsetExample]$
```
I am using GCC 7.4.1 compiler

**Comment by arturoc** [comment]
this seems to be a problem with the latest cairo packages, we are getting this error in the msys2 tests as well. can you post the output of `ls /usr/include/cairo`

**Comment by DarkStar1997** [comment]
This is it:
```
[rohan@rohan-pc ~]$ ls /usr/include/cairo
cairo-deprecated.h  cairo-gobject.h  cairo-script-interpreter.h  cairo-tee.h      cairo-xlib-xrender.h
cairo-features.h    cairo-pdf.h      cairo-script.h              cairo-version.h  cairo-xlib.h
cairo-ft.h          cairo-ps.h       cairo-svg.h                 cairo-xcb.h      cairo.h
[rohan@rohan-pc ~]$ 
```

**Comment by arturoc** [comment]
not sure what's going on, i0ve just tried with latest archlinux and everything works fine, i'm on gcc 8.2.1 so you probably just need to update

**Comment by DarkStar1997** [comment]
Everything here is updated, and I have GCC 8.2 but I had changed the symlinks to gcc-7and g++-7 because CUDA 10 supports upto GCC 7. But since you say everything works fine....I will try with GCC 8.2. Did you try with the website package or the one from github???

**Comment by arturoc** [comment]
website nightly

**Comment by arturoc** [comment]
yes this is a problem with opencv 4 not being compatible with opencv 3 anymore, while we port the OF addons to make them compatible, i've compiled an archlinux pkg for opencv 3.4.4.

You'll need to remove opencv 4 first then install this package:

```
pacman -R opencv
pacman -U https://openframeworks.cc/files/opencv-3.4.4-1-x86_64.pkg.tar.xz
```

That should solve the problem by now but not sure if the package will get updated when updating the system and changes in the standard library might break the package and need a recompile

**Comment by DarkStar1997** [comment]
OK, I'll try this package. Coming to permanent fixes, even if I downgrade OpenCV from 4 to 3, is there any way to fix the Cairo problem?

**Comment by arturoc** [comment]
cairo is ok, it seems to fail to find it cause the whole command fails when opencv is not found but once you fix opencv it should find cairo without problem

**Comment by DarkStar1997** [comment]
I have built OpenCV 3.4.5 from source and everything works like a charm now!!! The problem was with OpenCV 4 not being backwards compatible with OpenCV 3. Thank you for creating such an amazing library. I am closing this issue now.

**Comment by olsn** [comment]
I'm having  the same issue, however I'm on windows (msys2/mingw32) and when I try to install opencv 3.4.4, pacman says `package missing required signature` - any hints?

**Comment by DarkStar1997** [comment]
You can try building OpenCV 3 from source using CMake. That's what worked for me.

**Comment by arturoc** [comment]
thanks, going to reopen though since this will become a problem in other distros as well any time soon so we need a more definitive fix

**Comment by xseignard** [comment]
Hello,
Same here with 0.10.1 on rapsberry pi 3/raspbian stretch lite.

**Comment by arturoc** [comment]
@xseignard that must be something else, OF 0.10.1 should work fine in raspbian. have you run the install dependencies scripts?

**Comment by oxillo** [comment]
@arturoc The merge of #6252 should fix this  (at least it fixes it on Msys2!). May be we should close it.

**Comment by arturoc** [comment]
yes thanks!

**Comment by rossanag** [comment]
I am having the same issue, it doesn't find cairo.h but if I put:

**Comment by rossanag** [comment]
@arturoc  I am facing the same issue with the nightly building version, it doesn't find cairo.h even though it is installed (checked: ls /usr/include/cairo)
I need opencv 4 for other purposes, is there another fix to address this issue? 
Thanks in advance.
Ubuntu 19.04 64 bits

**Comment by patriciogonzalezvivo** [comment]
Hi! I got the same issue (I haven't use OF in a while)
```
ameworks/addons/ofxFaceTracker2/example-pose/src/main.cpp
In file included from /home/patricio/Desktop/openFrameworks/libs/openFrameworks/ofMain.h:65,
                 from /home/patricio/Desktop/openFrameworks/addons/ofxFaceTracker2/example-pose/src/main.cpp:1:
/home/patricio/Desktop/openFrameworks/libs/openFrameworks/graphics/ofCairoRenderer.h:4:10: fatal error: cairo.h: No such file or directory
    4 | #include "cairo.h"
      |          ^~~~~~~~~

```

Any idea?

**Comment by murilopolese** [comment]
I'm getting the same. Did you get to solve this, @patriciogonzalezvivo?

```
openFrameworks/graphics/ofCairoRenderer.h:4:10: fatal error: cairo.h: No such file or directory
    4 | #include "cairo.h"
```

and I also have Cairo installed:

```
$ sudo apt install libcairo2-dev
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
libcairo2-dev is already the newest version (1.16.0-5ubuntu2).
0 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.
```

**Comment by mak-simgithub** [comment]
hi, I am now also facing this issue, is there any way to fix this? I'm on OpenCV version 4.6
